### PR TITLE
[fix][cli] Fix start cli error

### PR DESCRIPTION
**Pull request by coderzc**
### Motivation 
Run `bin/hugegraph start` error

```bash
Starting HugeGraphServer...
bin/hugegraph: line 68: /start-hugegraph.sh: No such file or directory
Failed to start service
```

### Modifications

Add check for `Variables `

### CODE REVIEW

**Review by zyxxoo — APPROVED** [review]

File: hugegraph-dist/src/assembly/static/bin/hugegraph [hunk 1]
```diff
@@ -37,10 +37,19 @@
 ### END INIT INFO
 
 # Variables
-INSTALL_DIR=
```
Comment (imbajin) [inline hugegraph-dist/src/assembly/static/bin/hugegraph:40]: change the comment, it requires user to set a **fixed abs path** manually (should't use a relative path)

Comment (coderzc) [reply hugegraph-dist/src/assembly/static/bin/hugegraph:40]: You are right. and I think should add check for `Variables`

File: hugegraph-dist/src/assembly/static/bin/hugegraph [hunk 2]
```diff
@@ -37,6 +37,7 @@
 ### END INIT INFO
 
 # Variables
+# it requires user to set a fixed abs path manually
```
Comment (javeme) [inline hugegraph-dist/src/assembly/static/bin/hugegraph:40]: we can try `${INSTALL_DIR:?"Please setting variables 'INSTALL_DIR'"}`
for more details: https://stackoverflow.com/a/16753536

File: hugegraph-dist/src/assembly/static/bin/hugegraph [hunk 3]
```diff
@@ -41,13 +41,15 @@
 INSTALL_DIR=
 SERVER_PORT=
 
+${INSTALL_DIR:?"Please setting variables 'INSTALL_DIR'"}
```
Comment (imbajin) [inline hugegraph-dist/src/assembly/static/bin/hugegraph:44]: 😺

**Review by javeme — APPROVED** [review]

**Review by imbajin — APPROVED** [review]
I'll enhance this script later

**Review by javeme — APPROVED** [review]

File: hugegraph-dist/src/assembly/static/bin/hugegraph (hunk 2 above)
Comment (imbajin) [reply hugegraph-dist/src/assembly/static/bin/hugegraph:40]: > https://stackoverflow.com/a/16753536
@javeme seems a little strange, could it use directly or test it in local evn? (or should in a if condition?)

image

and it alert:
image

failed all cases in my ubuntu21

**Pull request merged**